prediction_market_agent/agents/microchain_agent/omen_functions.py

```diff
@@ -33,5 +40,36 @@ def __call__(self) -> str:
         )
 
 
+class CreatePredictionMarket(Function):
+    # Hard-coded low value before it's tested out more and real use-case is required.
+    INITIAL_FUNDS = xdai_type(0.01)
+
+    @property
+    def description(self) -> str:
+        return f"Use this function to create a new prediction market on Omen. Question of the prediciton market can only be binary, in the Yes/No format. Using this function will cost you {CreatePredictionMarket.INITIAL_FUNDS} xDai."
```

Comment:
Maybe add to the prompt the fact that the market should be valid (e.g. no violence, etc)? As described here https://github.com/gnosis/prediction-market-agent-tooling/blob/d770563c51f8d34a83f460c04efd68235c5278d1/prediction_market_agent_tooling/tools/is_invalid.py#L22

Reply:
Good idea (but I'd hope that LLM won't be violent in the nature 😄 )